stablehlo/reference/Ops.cpp

```diff
@@ -1215,6 +1219,21 @@ Tensor evalImagOp(const Tensor &operand, ShapedType resultType) {
   return result;
 }
 
+SmallVector<InterpreterValue> evalInfeedOp(Token token, Process *process,
+                                           Region &region, Scope &scope) {
+  if (!process)
+    llvm::report_fatal_error(
+        "infeed is only supported when run via interpreter.run_parallel");
+
+  auto mnemonic = process->infeed();
+  auto results = eval(region.getParentOfType<ModuleOp>()
+                          .lookupSymbol<func::FuncOp>(mnemonic)
+                          .getBody(),
+                      {}, process, &scope);
```

Comment:
Is there anything we should verify about these functions? Does a function with multiple return values work? Does each infeed function require that the function returns the same number of results? Will this error gracefully?

Reply:
Yes, we should add some checks. Here are the verifications I've added:
1. function should return 1 value
2. function return type should be a tensor
3. infeed attribute should be null or non empty.